WordPress/Classes/Utility/WebViewController/CookieJar.swift

```diff
@@ -4,7 +4,7 @@ import WebKit
 /// Provides a common interface to look for a logged-in WordPress cookie in different
 /// cookie storage systems.
 ///
-@objc protocol CookieJar {
+@objc public protocol CookieJar {
```

Comment:
I don't think you need it in `WordPressData`, do you?

Reply:
I'll double check in https://github.com/wordpress-mobile/WordPress-iOS/pull/24378.

I made these changes as a result of some code not seeing `CookieJar` when building Keystone the cherry picked of https://github.com/wordpress-mobile/WordPress-iOS/pull/24378, but it's of course possible I made a mistake elsewhere or there was caching involved. 

I'll let you know in my tomorrow 👀

Reply:
@kean now that I come back to your comment in the morning instead of late in my evening I see that the question of target membership was for WordPressData, but I read it as for Keystone.

In that case, rest assured, the protocol is not part of WordPressData, only Keystone (and WordPress and Jetpack till the compilation with Keystone as a dependency is sorted).

<img width="259" alt="image" src="https://github.com/user-attachments/assets/833ea4ca-12ba-4953-8c09-fefdd8abc28f" />

Reply:
In that case, it shouldn't need `public` modifiers, but they won't hurt.

Reply:
Mmm... I reverted this in https://github.com/wordpress-mobile/WordPress-iOS/pull/24378 and only got these build failures

![image](https://github.com/user-attachments/assets/1431c02c-69d1-468e-bd47-791c226a0a82)

I'll rebase then...

Reply:
It will need this to be merged https://github.com/wordpress-mobile/WordPress-iOS/pull/24399.